orbit/pkg/installer/installer.go

```diff
@@ -138,9 +142,17 @@ func (r *Runner) run(ctx context.Context, config *fleet.OrbitConfig) error {
 				continue
 			}
 		}
-		if err := r.OrbitClient.SaveInstallerResult(payload); err != nil {
+		err = retry.Do(func() error {
+			if err := r.OrbitClient.SaveInstallerResult(payload); err != nil {
+				log.Debug().Err(err).Msg("failed to save installer result, attempting retry")
```

Comment:
```suggestion
				log.Debug().Err(err).Msg("failed to save installer result, attempt #" + attemptNumber)
```

and then start at 1 and increment before `return err`, since if we get up to attempt 5 it's not gonna retry